docs/Where/project_resources.md

```diff
@@ -18,7 +18,15 @@
 
 [https://microsoft.github.io/mu/](https://microsoft.github.io/mu/)
 
+## Collaborate
+
+Feel free to submit [issues](https://github.com/Microsoft/mu/issues) to the central Mu repo, or Pull Requests to their corresponding code repo.
+

```

Comment:
I think it could be worded better.  something like 
Or, for faster communication and deeper collaboration, discuss on our team channel (request to join).

Reply:
Incorporated feedback.